pandas/tests/generic/test_finalize.py

```diff
@@ -784,3 +783,28 @@ def test_finalize_frame_series_name():
     df = pd.DataFrame({"name": [1, 2]})
     result = pd.Series([1, 2]).__finalize__(df)
     assert result.name is None
+
+
+def test_concat_retain_attrs():
+    '''Only retain the attrs when the attrs are the same across all
```

Comment:
Can you please fix the docstrings?
They need to be in triple quotes and the summary must fit in one line.
If additional lines are necessary, then they can be added as well, separated by an empty line from the summary line.